controls/SV-257258.rb

```diff
@@ -1,40 +1,43 @@
 control 'SV-257258' do
-  title 'RHEL 8 must terminate idle user sessions.'
+  title 'RHEL 8.7 and higher must terminate idle user sessions.'
   desc 'Terminating an idle session within a short time period reduces the window of opportunity for unauthorized personnel to take control of a management session enabled on the console or console port that has been left unattended.'
-  desc 'check', 'Verify that RHEL 8 logs out sessions that are idle for 15 minutes with the following command:
+  desc 'check', 'Note: This requirement applies to RHEL versions 8.7 and higher. If the system is not RHEL version 8.7 or newer, this requirement is not applicable.
 
-     $ sudo grep -i ^StopIdleSessionSec /etc/systemd/logind.conf
+Verify that RHEL 8 logs out sessions that are idle for 10 minutes with the following command:
```

Comment:
STIG revision history: "RHEL-08-020035 - Updated Rule Title, removed tmux workaround note from Fix Text, and added a note to the Check Text." I see the input updated in the inspec.yml file and the update for the version check. :+1: